core/common/src/LightSettings.ts

```diff
@@ -24,13 +24,17 @@ function extractIntensity(value: number | undefined, defaultValue: number) {
 export interface SolarLightProps {
   /** Intensity of the light, typically in [0..1] but can range up to 5. Default: 1.0 */
   intensity?: number;
-  /** Direction of the light in world coordinates. Default: (-1, -1, -1) */
+  /** Direction of the light in world coordinates. Default: (0.272166, 0.680414, 0.680414) */

```

Comment:
what do these numbers mean?

Reply:
This was our ancient default directional light direction from DgnClientSdk days, before we made the lighting configurable.
I can add an explanation.

Reply:
This is just some random light direction - not sure where it came from originally but it provides a decentt shadows in absence of a real direction.

Reply:
translated numbers to English.